.github/workflows/swift.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: macos-13
     strategy:
       matrix:
-        xcode: ['14.3']
+        xcode: ['15.1']
```

Comment:
is this change related?

Reply:
Yeah, since the breaking code is related to a new swift version, he had to update the runner XCode version to access that new swift version.